whistleblower/helpers/crop.py

```diff
@@ -0,0 +1,92 @@
+import sys
+
+import cv2
+import numpy
+
+TEXT_MIN_WIDTH = 35
+TEXT_MIN_HEIGHT = 10
+
+DEFAULT_WIDTH  = 850
+DEFAULT_HEIGHT = 1100
+
+KERNEL_WIDTH  = 25
+KERNEL_HEIGHT = 15
+
+
+def remove_borders(image, threshold, max_width, max_height):
+    height, width = image.shape[:2]
+
+    for i in range(max_width):
+        total = image[:, i].sum() / 255
+        if total > threshold:
+            image[:, i] = numpy.ones(height) * 255
+
+        total = image[:, width - i - 1].sum() / 255
+        if total > threshold:
+            image[:, i - 1] = numpy.ones(height) * 255
```

Comment:
We could extract this logic into a helper function `_remove_border(img, size, max_size, threshold)` in the DRY philosophy. So `remove_borders` could be simpler and we could document further what happens in `_remove_border`:

```python
def remove_borders(image, threshold, max_width, max_height):
    height. width, *_ = image.shape
    image = _remove_border(image, width, max_width, threshold)
    image = _remove_border(image, height, max_height, threshold)
    return image
```

And in `_remove_border` you could document the rational underneath it ; )

Reply:
The indexes is different. The first loop look at columns, the other looks at lines. To break in one function, the code should rotate the image and this will be a lot weird. I can refactor to create a remove_column_borders and remove_line_borders ou something like that.

Reply:
True story! I missed that, my bad. Just ignore my comment then : )